src/components/carousel/carousel.component.tsx

```diff
@@ -188,28 +206,8 @@ const BaseCarousel = ({
   );
 };
 
-BaseCarousel.propTypes = {
-  /** [legacy] Custom className */
-  className: PropTypes.string,
-  /** The selected tab on page load */
-  initialSlideIndex: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
-  /** The selected slide */
-  slideIndex: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
-  /** Individual tabs */
-  children: PropTypes.oneOfType([PropTypes.array, PropTypes.object]),
-  /** Enables the slide selector */
-  enableSlideSelector: PropTypes.bool,
-  /** Enables the previous button */
-  enablePreviousButton: PropTypes.bool,
-  /** Enables the next button */
-  enableNextButton: PropTypes.bool,
-  /** Action to be called on slide change */
-  onSlideChange: PropTypes.func,
-};
-
-const Carousel = withTheme(BaseCarousel);
 Carousel.displayName = "Carousel";
 
-export default BaseCarousel;
+export default Carousel;
 
-export { Carousel, Slide };
+export { Slide };
```

Comment:
Instead of importing and exporting the `Slide` component here, could we do this in the `index` file for `Carousel` instead to reduce the number of import/exports?